test/operations.py

```diff
@@ -0,0 +1,78 @@
+import requests, json
+
+class Operations:
+    @staticmethod
+    def create_index(endpoint, index_name, auth=None, data=None):
+        response = requests.put(f'{endpoint}/{index_name}', auth=auth)
+        if response.status_code != 200:
+            print('Failed to create index')
+            print(response.text)
+        else:
+            print('Created index successfully')
+            print(response.text)
+
+        pass
+
+    @staticmethod
+    def delete_index(endpoint, index_name, auth=None, data=None):
+        response = requests.delete(f'{endpoint}/{index_name}', auth=auth)
+        if response.status_code != 200:
```

Comment:
I recommend using this library for status codes: https://docs.python.org/3/library/http.html#http.HTTPStatus